code/modules/reagents/chemistry_properties/prop_positive.dm

```diff
@@ -773,13 +773,21 @@
 	durationmod_per_level = -0.1
 	radiusmod_per_level = -0.01
 
+	var/ignite_threshold = 4
+
 /datum/chem_property/positive/fire/oxidizing/reaction_mob(mob/M, method = TOUCH, volume, potency = 1)
 	var/mob/living/L = M
 	if(istype(L) && method == TOUCH)//Oxidizing 6+ makes a fire, otherwise it just adjusts fire stacks
 		L.adjust_fire_stacks(max(L.fire_stacks, volume * potency))
-		if(potency > 4)
+		if(potency > /datum/chem_property/positive/fire/oxidizing::ignite_threshold)
```

Comment:
Is there a reason to do `/datum/chem_property/positive/fire/oxidizing::ignite_threshold` instead of `initial(ignite_threshold)` or even just `ignite_threshold`?

Reply:
this syntax is analysed at compile time. it's a poor mans scoped define, namespaced under a specific datum